app/jobs/create_default_admin_set_job.rb

```diff
@@ -0,0 +1,6 @@
+class CreateDefaultAdminSetJob < ActiveJob::Base
+  def perform(account)
+    AccountElevator.switch!(account.tenant)
```

Comment:
Does the `ActiveJobTenant` (that automatically sets the tenant) not work here? If so, can we update it (or the way we build the job)?